tests/auto_inject/test_auto_inject_install.py

```diff
@@ -75,6 +74,15 @@ def test_profiling(self, virtual_machine):
         self._test_install(virtual_machine, profile=True)
 
 
+@features.crashtracking
```

Comment:
Do we also want `@features.ssi_crashtracking` ?

or maybe that instead of `@features.crashtracking`?

Trying to think how we'll tie this to the reporting we need to do, the `ssi_crashtracking` feature has both "ssi" and "crashtracking" tags.


also 👍🏻 for adding this but as `@missing_feature`, that is smart.